server/game/gamesteps/actionwindow.js

```diff
@@ -22,9 +22,9 @@ class ActionWindow extends PlayerOrderPrompt {
 
     activePrompt() {
         return {
-            menuTitle: 'Any actions or reactions?',
+            menuTitle: 'Initiate an action, or pass',
```

Comment:
I don't think we need to explicitly call out "or pass" here. I'd either title it just "Initiate an action" or "Any actions?", not particular on which.